### PR TITLE
Add description to each page (#1639)

### PR DESCRIPTION
**Pull request by rehmansheikh222**
Added concise descriptions to each page in the en folder.

### CODE REVIEW

File: _posts/2024-07-16-welcome-post.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
-title: Welcome to The Express Blog! 
+title: Welcome to The Express Blog!
+description: Introducing the new Express blog—a primary platform for announcements, updates, and communication from the Express technical committee.
```
Comment (carlosstenzel) [inline _posts/2024-07-16-welcome-post.md]: Introducing the new Express `blog — a` primary platform for announcements, updates, and communication from the Express technical committee.

File: en/3x/api.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: 3x-api
 title: Express 3.x - API Reference
+description: Access the API reference for Express.js version 3.x, noting that this version is end-of-life and no longer maintained; includes details on modules and methods.
```
Comment (carlosstenzel) [inline en/3x/api.md]: maint`ained - inclu`des details on mod

File: en/4x/api.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: 4x-api
 title: Express 4.x - API Reference
+description: Access the comprehensive API reference for Express.js 4.x, detailing all modules, methods, and properties for building web applications with this version.
```
Comment (carlosstenzel) [inline en/4x/api.md]: Access the API reference for Express.js 4.x, detailing all modules, methods, and properties for building web applications with this version.

File: en/5x/api.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: 5x-api
 title: Express 5.x - API Reference
+description: Access the comprehensive API reference for Express.js 5.x, detailing all modules, methods, and properties for building web applications with this latest version.
```
Comment (carlosstenzel) [inline en/5x/api.md]: Access the API reference for Express.js 5.x, detailing all modules, methods, and properties for building web applications with this latest version.

File: en/api.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: 4x-api
 title: Express 4.x - API Reference
+description: Access the comprehensive API reference for Express.js 4.x, detailing all modules, methods, and properties for building web applications with this version.
```
Comment (carlosstenzel) [inline en/api.md]: Access the API reference for Express.js detailing all modules, methods, and properties for building web applications with this version.

File: en/guide/routing.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: page
 title: Express routing
+description: Learn how to define and use routes in Express.js applications, including route methods, route paths, parameters, and using express.Router for modular routing.
```
Comment (carlosstenzel) [inline en/guide/routing.md]: Learn how to define and use routes in Express.js applications, including route methods, route paths, parameters, and using Router for modular routing.

File: en/starter/static-files.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 ---
 layout: page
 title: Serving static files in Express
+description: Understand how to serve static files like images, CSS, and JavaScript in Express.js applications using the built-in express.static middleware.
```
Comment (carlosstenzel) [inline en/starter/static-files.md]: Understand how to serve static files like images, CSS, and JavaScript in Express.js applications using the built-in 'static' middleware.

**Comment by carlosstenzel** [comment]
We would have to add a description page in other languages, even if it is in English.

**Comment by carlosstenzel** [comment]
before
Captura de Tela 2024-10-28 às 19 01 51

after
Captura de Tela 2024-10-28 às 19 10 30

**Comment by rehmansheikh222** [comment]
> We would have to add a description page in other languages, even if it is in English.

Hi @carlosstenzel , can you please elaborate what you mean by that? Does this mean adding description tag (like I did in the en folder) in all other language folders? @bjohansebas , please let me know your thoughts

**Comment by rehmansheikh222** [comment]
> before Captura de Tela 2024-10-28 às 19 01 51
> 
> after Captura de Tela 2024-10-28 às 19 10 30


Hi @carlosstenzel , do you want me to remove description tag from /en/blog/posts.md file?

**Comment by chrisdel101** [comment]
@carlosstenzel For the blog posts, the description was in the original page design but was removed before it was released. 
It was originally removed since it did not look good in the posts themselves, so I don't think we'd want to add it to the markup. 
Is this only for use in outside sites? That would seem okay, like for SEO, etc?

**Comment by carlosstenzel** [comment]
I looked at the metadata, and I didn't find the description
Captura de Tela 2024-10-29 às 19 27 01

but when searching on google. I was confused.
Captura de Tela 2024-10-29 às 19 28 11

** the description tag would be for better SEO

**Comment by carlosstenzel** [comment]
> > before Captura de Tela 2024-10-28 às 19 01 51
> > after Captura de Tela 2024-10-28 às 19 10 30
> 
> Hi @carlosstenzel , do you want me to remove description tag from /en/blog/posts.md file?

I added the images just for viewing and analysis by others. tks

**Comment by bjohansebas** [comment]
> >   We would have to add a description page in other languages, even if it is in English.
>
>Hi @carlosstenzel , can you please elaborate what you mean by that? Does this mean adding description tag (like I did in the en folder) in all other language folders? @bjohansebas , please let me know your thoughts

@rehmansheikh222 please do it, exactly as you said—doing the same in the other folders

**Comment by rehmansheikh222** [comment]
> > > We would have to add a description page in other languages, even if it is in English.
> > 
> > 
> > Hi @carlosstenzel , can you please elaborate what you mean by that? Does this mean adding description tag (like I did in the en folder) in all other language folders? @bjohansebas , please let me know your thoughts
> 
> @rehmansheikh222 please do it, exactly as you said—doing the same in the other folders

Hi @bjohansebas , just one last thing, does the description need to be in english (for other languages folder)?

**Comment by bjohansebas** [comment]
@rehmansheikh222 Yes, in English

**Comment by rehmansheikh222** [comment]
> @rehmansheikh222 Yes, in English

@carlosstenzel please check and let me know if there is anything else you would like me to change/add

**Comment by carlosstenzel** [comment]
> > @rehmansheikh222 Yes, in English
> 
> @carlosstenzel please check and let me know if there is anything else you would like me to change/add

I don't think it's ok ;)

**Comment by rehmansheikh222** [comment]
> > > @rehmansheikh222 Yes, in English
> > 
> > 
> > @carlosstenzel please check and let me know if there is anything else you would like me to change/add
> 
> I don't think it's ok ;)

Can you please elaborate?

**Comment by carlosstenzel** [comment]
> > > > @rehmansheikh222 Yes, in English
> > > 
> > > 
> > > @carlosstenzel please check and let me know if there is anything else you would like me to change/add
> > 
> > 
> > I don't think it's ok ;)
> 
> Can you please elaborate?

sorry

*** I think it's ok

**Comment by rehmansheikh222** [comment]
> > > > > @rehmansheikh222 Yes, in English
> > > > 
> > > > 
> > > > @carlosstenzel please check and let me know if there is anything else you would like me to change/add
> > > 
> > > 
> > > I don't think it's ok ;)
> > 
> > 
> > Can you please elaborate?
> 
> sorry
> 
> *** I think it's ok

Oh, okay. So, do you think you can approve the PR now?

**Review by carlosstenzel — APPROVED** [review]

**Comment by rehmansheikh222** [comment]
> > > > > > @rehmansheikh222 Yes, in English
> > > > > 
> > > > > 
> > > > > @carlosstenzel please check and let me know if there is anything else you would like me to change/add
> > > > 
> > > > 
> > > > I don't think it's ok ;)
> > > 
> > > 
> > > Can you please elaborate?
> > 
> > 
> > sorry
> > *** I think it's ok
> 
> Oh, okay. So, do you think you can approve the PR now?

@carlosstenzel it still says one review is required for the merging :)

**Comment by carlosstenzel** [comment]
@bjohansebas  @crandmck  @UlisesGascon  could you help?

**Review by bjohansebas — APPROVED** [review]
LGTM

**Comment by bjohansebas** [comment]
@rehmansheikh222 thanks

**Comment by rehmansheikh222** [comment]
> @rehmansheikh222 thanks

I'm glad I could contribute in some way, this was my first issue for open source and I plan to contribute more. Thanks all of you guys, you were really helpful.

**Comment by bjohansebas** [comment]
@rehmansheikh222 we have a documentation group, maybe you want to join and help us. see #1506

**Comment by rehmansheikh222** [comment]
> @rehmansheikh222 we have a documentation group, maybe you want to join and help us. see #1506

Yes sure, I will keep an eye on it. Thanks bro

**Pull request merged**